### PR TITLE
Tests that `newWorkspace` creates a new workspace

### PR DESCRIPTION
**Pull request by maximn**

### CODE REVIEW

File: javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java [hunk 1]
```diff
@@ -72,6 +73,17 @@ public void scratchFileShouldCreateFileAndWorkspaceContentsContainThatFile() thr
     org.hamcrest.MatcherAssert.assertThat(actualScratchFileContent, is(optionalWithValue(equalTo(content))));
   }
 
+  @Test
+  public void newWorkspaceCreatesANewWorkspace() throws IOException {
+      String path = "somePathForNewWorkspace";
```
Comment (ittaiz) [inline javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java]: these two lines are probably redundant right?

File: javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java [hunk 2]
```diff
@@ -72,6 +73,17 @@ public void scratchFileShouldCreateFileAndWorkspaceContentsContainThatFile() thr
...
+      scratchFile(path, "somecontent");
+
+      newWorkspace();
+
+      Optional<String> fullPath = findPath(workspaceContents(), path);
+      org.hamcrest.MatcherAssert.assertThat(fullPath, is(emptyOptional()));
```
[the remark this answers is not in the document]
Comment (maximn) [reply javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java]: yes

File: javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java (hunk 1 above)
Comment (ittaiz) [reply javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java]: but you're not creating it anywhere.
Also maybe rename the test to say `...CreatesANewCleanWorkspace`?

Comment (maximn) [reply javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java]: Renamed the test.
WDYM not creating it?
1. scratch file
2. create a new workspace
3. list files and make sure the file isn't there

**Comment by maximn** [comment]
synced to master

**Comment by ittaiz** [comment]
CLA bot is mistaken. Only commits by author.
@philwo strangely enough other PRs went fine when I asked the author to “merge from master” via the UI but this one didn’t. Maybe this data point can help the CLA bot maintainers

**Comment by philwo** [comment]
Thanks @ittaiz - I've asked our CLA bot people what might have caused this.

**Pull request merged**